### PR TITLE
Working processor action in servomixer

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/kinematics/Action.java [hunk 1]
```diff
@@ -57,4 +57,11 @@ public static Action createGestureToAction(String gestureName) {
     action.value = gestureName;
     return action;
   }
+
+  public static Action createProcessingAction(String methodName) {
```
Comment (AutonomicPerfectionist) [inline src/main/java/org/myrobotlab/kinematics/Action.java:69]: Java doc would be good

File: src/main/java/org/myrobotlab/service/Py4j.java [hunk 2]
```diff
@@ -235,17 +236,19 @@ public void connectionStopped(Py4JServerConnection gatewayConnection) {
    * 
    * @param code The Python code to execute in the interpreter.
    */
-  public void exec(String code) {
+  public boolean exec(String code) {
```
Comment (AutonomicPerfectionist) [inline src/main/java/org/myrobotlab/service/Py4j.java:240]: Need `@Override` if it's part of an interface now

File: src/main/java/org/myrobotlab/service/Python.java [hunk 3]
```diff
@@ -49,7 +50,7 @@
  * @author GroG
  * 
  */
-public class Python extends Service<PythonConfig> implements ServiceLifeCycleListener, MessageListener {
+public class Python extends Service<PythonConfig> implements ServiceLifeCycleListener, MessageListener, Processor {
```
Comment (AutonomicPerfectionist) [inline src/main/java/org/myrobotlab/service/Python.java:53]: The exec method should have Override annotation

File: src/main/java/org/myrobotlab/service/interfaces/Processor.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package org.myrobotlab.service.interfaces;
+
+public interface Processor {
```
Comment (AutonomicPerfectionist) [inline src/main/java/org/myrobotlab/service/interfaces/Processor.java:3]: Java doc here explaining what this is used for would be good

**Review by AutonomicPerfectionist — CHANGES_REQUESTED** [review]

File: src/main/java/org/myrobotlab/kinematics/Action.java (hunk 1 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/kinematics/Action.java:69]: updated

File: src/main/java/org/myrobotlab/service/Python.java (hunk 3 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/Python.java:53]: updated

File: src/main/java/org/myrobotlab/service/interfaces/Processor.java (hunk 4 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/interfaces/Processor.java:3]: updated

**Pull request merged**